redisbeat.yml

```diff
@@ -6,7 +6,7 @@ input:
   #period: 10
 
   # Host of redis server, default is localhost
-  #host: "localhost"
+  host: "localhost"
```

Comment:
Why are we uncommenting these? Do the defaults in code (`redisbeat.config()`) not work anymore?